cylc/doc/etc/tutorial/retries-tutorial/suite.rc

```diff
@@ -10,14 +10,14 @@
     [[win]]
     [[roll_doubles]]
         script = """
-sleep 10
-RANDOM=$$  # Seed $RANDOM
-DIE_1=$((RANDOM%6 + 1))
-DIE_2=$((RANDOM%6 + 1))
-echo "Rolled $DIE_1 and $DIE_2..."
-if (($DIE_1 == $DIE_2)); then
-   echo "doubles!"
-else
-   exit 1
-fi
-"""
+            sleep 10
+            RANDOM=$$  # Seed $RANDOM
+            DIE_1=$((RANDOM%6 + 1))
+            DIE_2=$((RANDOM%6 + 1))
+            echo "Rolled $DIE_1 and $DIE_2..."
+            if (($DIE_1 == $DIE_2)); then
+               echo "doubles!"
+            else
+               exit 1
+            fi
+        """
```

Comment:
> Hmmm, I don't recall that we agreed on this indentation thing.

😁

Besides readibility issues there was a lot of inconsistency with indentation and code examples weren't always consistent with the suite design guide.

> even though I'm obviously right about it!!!

🤮

> But we should at least put a comment in the user guide to explain why it is OK to indent the lines of a bash script

True, there is a note in the suite design guide to that effect but we should probably add a note to the tutorial and writing suites sections if not already present.

> and when it occasionally might not work.

Since the dedent change indenting scripts works as expected even with indentation-sensitive stuff like heredocs or Python scripts:

```console
$ cat x/suite.rc
[scheduling]
    [[graph]]
        R1 = foo
[runtime]
    [[foo]]
        script = """
            cat >> log.txt <<_EOF
            a
            b
            _EOF
        """
$ cat x/work/1/foo/log.txt 
a
b
```

Are there any remaining cases where it can fail?

Reply:
> Are there any remaining cases where it can fail? 

Meh, I can't think of any.  Here docs are the obvious problem for bash job scripts, but dedent solved that.  

https://github.com/cylc/cylc-doc/issues/149